scheduler-jobs/deploy_data_specific_image

```diff
@@ -9,6 +9,9 @@
 # - repo_url: path to the github repo (ex.: https://github.com/crim-ca/pavics-jupyter-images)
 #   branch: name of the branch containing the required version of the data
 #   source_path: path of the desired file or folder in the source repo
+#   dest_sub_dir: Data is sent to a tutorial-notebooks directory with the same name as the image running the script.
+#            Use this parameter to specify a subdirectory path (without a prefix '/') in this directory where the data will be sent.
+#            If the parameter is absent or left empty, data is sent directly to the image's tutorial-notebooks folder
```

Comment:
So before this `dest_sub_dir` fix, given `crim-eo` image, the notebooks would have been deployed at the root `/data/jupyterhub_user_data/tutorial-notebooks/` instead of `/data/jupyterhub_user_data/tutorial-notebooks/crim-eo`?

Reply:
No the host path mount inside the image is /data/jupyterhub_user_data/tutorial-notebooks/<image_name> so for your example /data/jupyterhub_user_data/tutorial-notebooks/crim-eo/
The sub_dir only allows to checkout multiple repository, for exemple : 
common notebooks with a dest_sub_dir "common" will be checkout to /data/jupyterhub_user_data/tutorial-notebooks/crim-eo/common
eo notebooks with a dest_sub_dir "eo" will be checkout to /data/jupyterhub_user_data/tutorial-notebooks/crim-eo/eo
If dest_sub_dir stay empty they will be checkout to /data/jupyterhub_user_data/tutorial-notebooks/crim-eo/

Reply:
Ah I see, thanks.